routing.go

```diff
@@ -110,6 +114,10 @@ type RecvdVal struct {
 
 // GetValue searches for the value corresponding to given Key.
 func (dht *IpfsDHT) GetValue(ctx context.Context, key string, opts ...routing.Option) (_ []byte, err error) {
+	if !dht.enableValues {
```

Comment:
is this correct behaviour? We might know a peer that supports `GetValue` and so on.